ideascube/serveradmin/tests/test_models.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime, timezone
+
+import freezegun
+import pytest
+
+
+@pytest.mark.parametrize(
+    'value, type_method',
+    [
+        (True, 'as_boolean'),
+        (42, 'as_integer'),
+        (None, None),
+        ('A string', 'as_string'),
+    ],
+    ids=[
+        'boolean',
+        'int',
+        'none',
+        'string',
+    ])
+@pytest.mark.usefixtures('db')
+def test_settings(value, type_method, user):
+    from ideascube.serveradmin.models import Setting
+
+    fakenow = datetime.now(tz=timezone.utc)
+    assert Setting.objects.count() == 0
+
+    with freezegun.freeze_time(fakenow):
```

Comment:
I don't see it as a problem to have this new dep.

In fact, I see it as a basic need in order to write unit tests properly, just like pytest.

And in fact, I was thinking of slowly going back over all unit tests, and making them use freezegun where appropriate, one by one. (but not in this branch, of course)

What you suggest is a different test: it allows ensuring that if we explicitly set a date for settings, then it is stored and retrieved properly.

freezegun allows testing something else: it allows ensuring that the date is correctly set by `auto_now`.